lib/Tokenizer.js

```diff
@@ -635,6 +641,15 @@ Tokenizer.prototype.write = function(chunk){
 Tokenizer.prototype._parse = function(){
 	while(this._index < this._buffer.length && this._running){
 		var c = this._buffer.charAt(this._index);
+		// Detect Markdown code so that it is parsed as text instead of HTML
+		if(c === '`'){
+			this._isMarkdownCode = !this._isMarkdownCode;
+		} else if(c === '<'){
+			var nextChar = this._buffer.charAt(this._index + 1);
+			if(nextChar === '='){
+				this._isInequality = true;
+			}
+		}
```

Comment:
- Add a newline before and after this entire block.
- Maybe add a section name like the ones below.

Reply:
By section name, do you mean changing `'='` into something like `EQUALS`?

```
if(nextChar === EQUALS){
    this._isInequality = true;
}
```

Reply:
I mean these: https://github.com/MarkBind/htmlparser2/pull/1/files#diff-00550ec11d6b5101df5a54c5fee7cc2eR670

Reply:
Would `special conditions` be an appropriate section name?

Reply:
Fine for now.